inventory/sample/group_vars/k8s_cluster/k8s-cluster.yml

```diff
@@ -118,7 +118,7 @@ kube_apiserver_ip: "{{ kube_service_subnets.split(',') | first | ansible.utils.i
 kube_apiserver_port: 6443  # (https)
 
 # Kube-proxy proxyMode configuration.
-# Can be ipvs, iptables
+# Can be ipvs, iptables, nftables
 kube_proxy_mode: ipvs
```

Comment:
We should plane to change the default at some point and indicates so with a TODO:
https://kubernetes.io/blog/2025/02/28/nftables-kube-proxy/#future-plans
the IPVS mode is apparently not the future.
From the blog paste it does not look like the upgrade path is painful.

Reply:
Good Idea, We can change the default followed by the upstream: https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-proxy/app/server_linux.go#L64 .